test/spec/controllers/eventRegistrations.spec.js

```diff
@@ -104,4 +104,28 @@ describe('Controller: eventRegistrations', function() {
     scope.registerUser();
     expect(openModal).toHaveBeenCalled();
   });
+
+  it('isGroupRegistrant should return true for primary user group registrations', function() {
```

Comment:
Can you change this to an arrow function as well?

Reply:
fixed